src/main/java/org/jitsi/videobridge/Conference.java

```diff
@@ -1164,13 +1144,14 @@ public Content getOrCreateContent(String name)
 
         synchronized (contents)
         {
-            for (Content aContent : contents)
+            content
+                = contents.stream()
+                    .filter(c -> c.getName().equals(name))
+                    .findAny().orElse(null);
```

Comment:
i think `findFirst` would be more consistent with the original logic here (though i know that it shouldn't really make a difference, since we only expect it to match one)